src/language-css/utils/index.js

```diff
@@ -26,24 +26,8 @@ const colorAdjusterFunctions = new Set([
   "hwba",
 ]);
 
-function getAncestorCounter(path, typeOrTypes) {
-  const types = Array.isArray(typeOrTypes) ? typeOrTypes : [typeOrTypes];
-
-  let counter = -1;
-  let ancestorNode;
-
-  while ((ancestorNode = path.getParentNode(++counter))) {
-    if (types.includes(ancestorNode.type)) {
-      return counter;
-    }
-  }
-
-  return -1;
-}
-
-function getAncestorNode(path, typeOrTypes) {
-  const counter = getAncestorCounter(path, typeOrTypes);
-  return counter === -1 ? null : path.getParentNode(counter);
+function getAncestorNode(path, type) {
```

Comment:
The same function is defined in `printer-markdown.js` too. I think we can refactor it https://github.com/prettier/prettier/blob/next/src/language-markdown/printer-markdown.js#L509-L527

Reply:
I know, it's not used simple as this one.